noaa.go

```diff
@@ -98,16 +98,18 @@ type ForecastResponsePeriod struct {
 
 // ForecastResponsePeriodHourly provides the JSON value for a period within an hourly forecast.
 type ForecastResponsePeriodHourly struct {
-	ID               int32  `json:"number"`
-	Name             string `json:"name"`
-	StartTime        string `json:"startTime"`
-	EndTime          string `json:"endTime"`
-	IsDaytime        bool   `json:"isDaytime"`
-	TemperatureTrend string `json:"temperatureTrend"`
```

Comment:
I would keep `TemperatureTrend` as it appears to be part of the current spec (can be null).